Node Typing Typescript Compile error, TS2304: Cannot find name 'IterableIterator'.

Using the latest Node Definition file `#6.0.0+20160830141956`, the compiler throws a `Typescript Compile error, TS2304: Cannot find name 'IterableIterator'.`  It is coming from a deprecated `NodeBuffer` method (It appears in there 3 times).  Is there another typing file I'm missing, or is this an issue?

I'm receiving this error with the `whatwg-fetch` types with TypeScript Version 2.0.2

```
{
  "name": "my-app",
  "globalDevDependencies": {
    "es6-shim": "registry:dt/es6-shim#0.31.2+20160602141504",
    "whatwg-fetch": "registry:dt/whatwg-fetch#0.0.0+20160829180742"
  }
}
```

```
typings/globals/whatwg-fetch/index.d.ts(19,16): error TS2304: Cannot find name 'IterableIterator'.
typings/globals/whatwg-fetch/index.d.ts(21,13): error TS2304: Cannot find name 'IterableIterator'.
typings/globals/whatwg-fetch/index.d.ts(22,15): error TS2304: Cannot find name 'IterableIterator'.
typings/globals/whatwg-fetch/index.d.ts(23,6): error TS2304: Cannot find name 'Symbol'.
typings/globals/whatwg-fetch/index.d.ts(23,26): error TS2304: Cannot find name 'IterableIterator'.
typings/globals/whatwg-fetch/index.d.ts(88,11): error TS2304: Cannot find name 'ReadableStream'.
```

--- a/node/node.d.ts
+++ b/node/node.d.ts
@@ -452,6 +452,8 @@ declare namespace NodeJS {
     }
 }
 
+interface IterableIterator<T> {}
+
 /**
  * @deprecated
  */

--- a/node/node.d.ts
+++ b/node/node.d.ts
@@ -452,7 +452,7 @@ declare namespace NodeJS {
     }
 }
 
-interface IterableIterator<T> {}
+interface IterableIterator<T> extends Array<T> {}
 
 /**
  * @deprecated

--- a/node/node.d.ts
+++ b/node/node.d.ts
@@ -452,7 +452,7 @@ declare namespace NodeJS {
     }
 }
 
-interface IterableIterator<T> extends Array<T> {}
+interface IterableIterator<T> {}
 
 /**
  * @deprecated

